### DISCUSSION TITLE
Azure SAML

### DISCUSSION THREAD

**Discussion by MacLeanIan** [opendcim/openDCIM#1363] [Q&A] [answered]
Hello,

I am trying to move my authentication from PingFed SAML to Azure. I have gotten the new Cert and metadata, but when attempting to authenticate, I am getting “AADSTS75011: Authentication method 'Unspecified' by which the user authenticated with the service doesn't match requested authentication method 'Password, ProtectedTransport'.”

 

Researching, I found https://learn.microsoft.com/en-us/troubleshoot/azure/active-directory/error-code-aadsts75011-auth-method-mismatch

 

Wondering, if anyone has received this error before and what steps were taken?

Thank you in advance

**Comment by samilliken** [comment, marked as answer]
I'm assuming that you haven't upgraded to 23.01 yet, where the default behavior is different for SAML.   See this [Commit](https://github.com/opendcim/openDCIM/commit/eb01d7f393acd224dbac38601179230bf0ae9abf) from 23.01 for what you need to change.
